datanode/sqlstore/migrations/0102_retro_fill_team_reward_totals.sql

```diff
@@ -0,0 +1,72 @@
+-- +goose Up
+
+-- Carry over all reward totals that are missing across epochs.
+INSERT INTO game_reward_totals (game_id, party_id, asset_id, market_id, team_id, total_rewards, total_rewards_quantum, epoch_id)
+    SELECT
+        grto.game_id,
+        grto.party_id,
+        grto.asset_id,
+        grto.market_id,
+        grto.team_id,
+        grto.total_rewards,
+        grto.total_rewards_quantum,
+        (epoch_id + 1) AS epoch_id
+    FROM game_reward_totals AS grto
+    WHERE grto.epoch_id < (SELECT MAX(id)-1 FROM epochs)
+    AND NOT EXISTS (
+        SELECT 1
+        FROM game_reward_totals AS grtc
+        WHERE grtc.party_id = grto.party_id
+        AND grtc.asset_id = grto.asset_id
+        AND grtc.market_id = grto.market_id
+        AND grtc.game_id = grto.game_id
+        AND grtc.team_id = grto.team_id
+        AND grtc.epoch_id = grto.epoch_id + 1
+    );
```

Comment:
I ran a quick test with this inside a transaction and I don't think this does what it's supposed to. I think it will only carry over missing transactions from one epoch, to the next, but not any further than that.

Also we would only want the stats carried over for the duration of the game, there's no point in carrying forward the stats if in the next epoch the game is over, or we will end up carrying this forward indefinitely.